src/main/java/adris/altoclef/butler/Butler.java

```diff
@@ -46,14 +46,12 @@ public Butler(AltoClef mod) {
 
         // Receive system events
         EventBus.subscribe(ChatMessageEvent.class, evt -> {
-            if (evt.messageType == MessageType.SYSTEM) {
```

Comment:
I understand why this was removed, but most servers use specifically SYSTEM and CHAT as whisper and chat channels respectively. I cannot see any reason to keep there here so I concur with the removal.